### ISSUE TITLE
Fail to reproduce eccc downscaling

### ISSUE THREAD

**Issue by whpy** [IBM/granite-wxc#16]
I am trying to reproduce the eccc downscaling example. I fine-tune the model following the file `eccc_downscaling_finetune.ipynb`. The configuration file I adopt for fine-tuning is `config_Unet.yaml`, where the learning rate (1.0e-5), batch size (1) and epoch nums (5) are not changed. The data are all downloaded from the URLs provided in the file README.md. The index files for datasets are generated by the given script `index_generator.py`.

While fine-tuning the model, I get the same output on console while executing the script `eccc_downscaling_finetune.ipynb` as illustrated in this file. Then I test performance of the last saved checkpoint model by executing `eccc_downscaling_inference.ipynb`. And I find that the power spectrum curve of my fine-tuned model (Fig. 1) does not fit the target as well as that shown in the file (Fig. 2).

I guess that it is because the hyperparameters I adopt is wrong. Could you please share the correct configuration (config_Unet.yaml) with me? Thank you!

<img width="906" height="550" alt="Image" src="https://github.com/user-attachments/assets/0ee64c4d-6c54-4080-8874-19a19f2411f2" />
Fig.1 The power spectrum curve of my fine-tuned model

<img width="1169" height="703" alt="Image" src="https://github.com/user-attachments/assets/ee402c67-a410-4d45-a093-c2a41f4c6ce7" />
Fig. 2 The power spectrum curve illustrated in `eccc_downscaling_inference.ipynb`
